RecipeMatch/Views/DraggableViewBackground.m

```diff
@@ -42,25 +43,35 @@ - (id)initWithFrame:(CGRect)frame
     
     if (self) {
         [super layoutSubviews];
-        [self setupView];
-        loadedCards = [[NSMutableArray alloc] init];
-        allCards = [[NSMutableArray alloc] init];
-        cardsLoadedIndex = 0;
-        [self loadCards];
+    
     }
     return self;
 }
+-(void)fetchRecipes{
+    
+    [[APIManager shared] getRecipes:self.preferences withCompletion: ^(NSMutableArray *recipes, NSError *error) {
+        if(recipes)
+        {
+            self.recipes = recipes;
+            [self getCards];
+        } else {
+            NSLog(@"😫😫😫 Error getting home timeline: %@", error.localizedDescription);
+        }
+    }];
```

Comment:
there shouldn't be any api calls in views, only in VCs.
you can load the recipes there, and then call a reload method inside the draggableViewBackground